apps/dashr-multipage-report/README.md

```diff
@@ -0,0 +1,44 @@
+# Dash Wind Streaming
+
+## About this app
+
+This app queries a SQL database every second and uses the data to update the wind speed diagram and the wind direction diagram. 
+The wind speed values are then binned in real time to generate the wind histogram plot.
+
+Original repo: [plotly/dash-wind-streaming](https://github.com/plotly/dash-wind-streaming)
+
+
+## How to run this app
+
+(The following instructions apply to Posix/bash. Windows users should check
```

Comment:
Can we change the instructions so that instead of `Posix/bash`, it reads "Linux and OS X" users?